app/pdf/generator.py

```diff
@@ -849,113 +847,11 @@ def generate_latex(atbd: Atbds, filepath: str, journal=False):  # noqa: C901
         ):
             continue
 
-        # SECTION KEY POINTS
-        if section_name in ["key_points"]:
-
-            pass
-
-        # SECTION INTRODUCTION
-        if section_name == "introduction":
-
-            pass
-
-        # SECTION CONTEXT BACKGROUND
-        # Note: This title section is handled elsewhere as logic is currently written
-
-        # SECTION HISTORICAL PERSPECTIVE
-        if section_name == "historical_perspective":
-
-            pass
-
-        # SECTION ADDITIONAL INFORMATION
-        if section_name == "additional_information":
-
-            pass
-
-        # SECTION ALGORITHM DESCRIPTION
-        if section_name == "algorithm_description":
-
-            pass
-
-        # SECTION ALGORITHM SECTION
-        # process key points as a text attribute
-        if section_name in [
-            "algorithm_input_variables_caption",
-            "algorithm_output_variables_caption",
-        ]:
-
-            pass
-
-        # SECTION PLAIN SUMMARY
-        if section_name == "plain_summary":
-
-            pass
-
-        #  SECTION KEYWORDS
-        if section_name == "keywords" and atbd_version.keywords:
-            # doc.append(Command("begin", arguments="itemize"))
-            # for keyword in atbd_version.keywords:
-            #     doc.append(Command("item", arguments=keyword["label"]))
-            # doc.append(Command("end", arguments="itemize"))
-
-            continue
-
-        # SECTION CONTACT
-        if section_name == "contacts":
-
-            pass
-
-        if not document_data.get(section_name):
-            doc.append(process(CONTENT_UNAVAILABLE))  # type: ignore
-            continue
```

Comment:
Just flagging if this is expected. My hunch is this is a duplicate, seeing the same code on the other PR.